packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts

```diff
@@ -197,15 +199,38 @@ export async function deploy(
   ctx.logProvider.info(getLocalizedString("core.deploy.startNotice", PluginDisplayName.Solution));
   const result = await executeConcurrently(thunks, ctx.logProvider);
 
+  const botTroubleShootLink =
+    "https://aka.ms/teamsfx-bot-help#how-can-i-troubleshoot-issues-when-teams-bot-isnt-responding-on-azure";
+  const botTroubleShootDesc = getLocalizedString("core.deploy.botTroubleShoot");
+  const botTroubleShootLearnMore = getLocalizedString("core.deploy.botTroubleShoot.learnMore");
   if (result.kind === "success") {
     if (inAzureProject) {
-      const msg = getLocalizedString("core.deploy.successNotice", ctx.projectSetting.appName);
-      ctx.logProvider.info(msg);
-      ctx.userInteraction.showMessage("info", msg, false);
+      let msg = getLocalizedString("core.deploy.successNotice", ctx.projectSetting.appName);
+      // Append a new sentence for bot trouble shootting.
+      if (inBotProject) {
+        msg += ` ${botTroubleShootDesc}`;
+      }
+      ctx.logProvider.info(msg + ` ${botTroubleShootLearnMore}: ${botTroubleShootLink}.`);
+      if (inBotProject) {
+        // Show a `Learn more` action button for bot trouble shooting.
+        ctx.userInteraction
```

Comment:
Is it possible to only add button in this step and then call showMessage together?
e.g.
```
if (!botMsg) {
  // add bot button
}
ctx.userInteraction.showMessage("info", msg, false);
```

Reply:
It seems not possible because of the API contract of `showMessage` as you can see how it apply the `Learn more` clicking action.